### DISCUSSION TITLE
Is there a chezmoi equivalent to `git checkout`

### DISCUSSION THREAD

**Discussion by agingorange** [twpayne/chezmoi#2866] [Q&A] [answered]
To undo changes, I go to the repo and do `git checkout <filename>` and `chezmoi apply -v`. Is there an equivalent chezmoi command?

**Comment by twpayne** [comment, marked as answer]
No, there is no equivalent chezmoi command.

You can create a shell function:

```bash
chezmoi-checkout() {
    chezmoi git checkout -- $(chezmoi source-path $*) && chezmoi apply -v $*
}
```

and then run

```console
$ chezmoi-checkout $filename
```

**Comment by agingorange** [comment]
Excellent! Thanks much.
